changelog/1063.bugfix.md

```diff
@@ -0,0 +1 @@
+Fix `App Instantiation Error` as a result of a `Sanic` upgrade.
```

Comment:
I think this change was part of upgrading sanic instead of a bug that we fixed. This changelog doesn't make a lot of sense for the customer